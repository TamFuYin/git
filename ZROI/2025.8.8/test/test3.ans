32
28
31
29
35
30
17
30
23
27
33
32
41
14
18
48
16
15
15
19
18
39
14
24
16
30
17
38
18
40
23
41
29
52
16
14
21
21
18
16
23
34
15
17
16
28
15
14
18
18
