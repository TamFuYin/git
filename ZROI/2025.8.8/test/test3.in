50 15 50
101111010101111
011111110011010
111110011011011
001000111000010
100001011000000
010001110011010
111010110101001
111111110100010
111110101111000
011111111000101
001110010000110
110011110100110
111110001110000
011011111100101
011100111011011
001000110001111
010101110001010
101001011001100
110111001101101
001000011100000
101011000001101
010110001011010
110111111011101
110100010001001
100011100001001
000111100100101
000110011100111
100010010101010
110100111001001
011111010111110
100000001001111
110000000100110
011010101010101
001110111101010
100010110000011
111001100010111
000010001011000
000111100011110
000000010011100
010100001101101
010010100101110
011111110111001
011000110101011
101011010100011
101010011110111
000100111001011
011110000011011
101111110010000
001101000000101
010010001101000
8 28
25 42
1 19
20 37
16 40
22 41
24 28
27 47
31 43
6 20
17 38
21 42
5 34
22 22
1 5
12 50
47 49
48 49
29 29
39 46
18 22
12 40
38 40
35 48
41 43
5 22
35 40
2 28
42 47
11 40
3 13
15 46
27 45
6 48
36 39
40 40
34 44
8 16
40 44
32 34
23 34
19 42
45 46
7 9
46 48
7 22
48 48
50 50
17 21
45 50
