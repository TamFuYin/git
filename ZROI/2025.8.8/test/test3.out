32
28
31
29
35
30
17
31
24
27
33
32
41
14
18
49
16
15
15
19
18
39
14
24
16
30
17
38
18
40
23
42
29
53
16
14
22
21
18
16
23
34
15
17
16
28
15
14
18
18
